reliability-engineering/pipelines/tasks/generate-roll-instances-pipeline.yml

```diff
@@ -83,8 +83,8 @@ run:
                   }
                 ]
               },
-              {
-                "task": "get-current-workers",
+              (.[] | select (.team != "main") | {
+                "task": ("get-current-workers-" + .team),
```

Comment:
I _think_ this will just create an array of `get-current-workers-foo` tasks in this position, leaving just a single instance of the other tasks dangling below. Suspect we will have to create this as a second array which we concatenate to the first.

Reply:
Oh and either we'll have to flatten the resulting array-of-arrays or use the `do:` step https://concourse-ci.org/jobs.html#do-step